EELS/EELS_elemental_mapping.ipynb

```diff
@@ -0,0 +1,789 @@
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ax_haadf.imshow(\n",
+    "    np.flipud(s_haadf.data),\n",
+    "    interpolation='nearest',\n",
+    "    origin='upper',\n",
+    "    extent=s_haadf.axes_manager.signal_extent,\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Disable axis ticks"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ax_haadf.set_xticks([])\n",
+    "ax_cu.set_xticks([])\n",
+    "ax_zn.set_xticks([])\n",
+    "ax_zn.set_yticks([])\n",
+    "ax_cu.set_yticks([])\n",
+    "ax_haadf.set_yticks([])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Scalebars can be added by using `AnchoredSizeBar`. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "fontprops = fm.FontProperties(size=18)\n",
+    "scalebar0 = AnchoredSizeBar(\n",
+    "        ax_cu.transData,\n",
+    "        5, '5 nm', 1, # length of bar, label, loc\n",
+    "        pad=0.1,\n",
+    "        color='white',\n",
+    "        frameon=False,\n",
+    "        size_vertical=0.6,\n",
+    "        fontproperties=fontprops,\n",
+    ")\n",
+    "scalebar1 = AnchoredSizeBar(\n",
+    "        ax_cu.transData,\n",
+    "        5, '5 nm', 1,\n",
+    "        pad=0.1,\n",
+    "        color='white',\n",
+    "        frameon=False,\n",
+    "        size_vertical=0.6,\n",
+    "        fontproperties=fontprops,\n",
+    ")\n",
+    "scalebar2 = AnchoredSizeBar(\n",
+    "        ax_cu.transData,\n",
+    "        5, '5 nm', 1,\n",
+    "        pad=0.1,\n",
+    "        color='white',\n",
+    "        frameon=False,\n",
+    "        size_vertical=0.6,\n",
+    "        fontproperties=fontprops,\n",
+    ")\n",
+    "ax_cu.add_artist(scalebar0)\n",
+    "ax_zn.add_artist(scalebar1)\n",
+    "ax_haadf.add_artist(scalebar2)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We add labels to indicate what is shown in each image."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ax_cu.text(0.05, 0.05, 'Cu', color='white', size=18, transform=ax_cu.transAxes)\n",
+    "ax_zn.text(0.05, 0.05, 'Zn', color='white', size=18, transform=ax_zn.transAxes)\n",
+    "ax_haadf.text(0.05, 0.05, 'HAADF', color='white', size=18, transform=ax_haadf.transAxes)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Then colorbars are added. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "cb_zn = fig.colorbar(ax_zn.images[0], cax=cbar_zn, extend='both', orientation='horizontal', label=\"Relative Zn, [a.u.]\")\n",
```

Comment:
Are these arbitrary units?

Reply:
I guess "non-calibrated" is the more correct term, as we don't know the beam current well enough to get quantitative information.

Reply:
For the elemental maps, the beam current is not necessary, because the cross section should be sensitivity factor, similarly to Cliff-Lorimer for EDS - there are similar to the Hartree Slater cross section in GMS.